.gitignore

```diff
@@ -1,2 +1,4 @@
 build
 node_modules
+src/test_files.js
+test/test_files.test.js
```

Comment:
remove these two from the gitignore since you should delete these files from the PR